text/0000-filesystem-integration.md

```diff
@@ -0,0 +1,136 @@
+
+*This will significantly lower the barrier to entry for new users to qri, especially GUI users, as they can quickly pull down published datasets and begin working with them without having to understand how IPFS/QRI stores and maintains the data.*  
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The intent is to make a qri dataset more tangible (and thus easier to get started with) by adding a common interface (the user's filesystem).  
+
+When a new user wants the files in a remote git repo, they use `git clone {repoUrl}` and a new directory is created on their filesystem containing those files.  They can use _whatever tooling they want_ to make changes to the files, and commit them locally when they reach a critical point.
+
+We want the qri dataset user to have a similar experience: `qri add /b5/world_bank_population/` should create a new directory called `world_bank_population` on the filesystem containing the data file and associated metadata. To change dataset, the user uses _whatever tooling they want_ to modify the files, then commits the changes.
+
+## One-to-One
+
+This is a 1-1 relationship, a dataset can only link to one directory on the filesystem.
+
+## Files
+
+The following files abstract the qri dataset model, providing well-named files for the most recognizable parts of the dataset.
+
+### `meta.json`
+
+The dataset's `meta` component.
+
+### `schema.json`
+
+The dataset's `structure.schema` component
+
+### `body.[csv|json|xlsx]`
+
+The data file!
+
+### `dataset.json`
+
+Any additional uncontrolled `dataset` fields.  This file will probably not be used by novice users.  Users who are more familiar with the dataset model can use this file to add advanced configuration.
+
+### `.qri-ref`
+
+An invisible file used to specify the link between the current directory and the Qri repo.
+
+## Commands
+
+- `qri init` - interactive prompt that sets up a qri dataset with linked files in the current directory. (User should have already created a directory to hold the dataset, `cd` into it, then run `qri init`)
+
+This should have an `non-interactive` flag that only needs a dataset name, which will link an empty dataset working directory to a name reference in the qri repo. (reserving the name for a future commit)
+
+### Prompts:
+  -  **Dataset Name** - (autopopulate with the name of the current directory) generates `meta.json` in the working directory with the supplied name as `title` and boilerplate `description`, `keywords`, etc
+
+  -  **File Type** - csv/json? Based on the user's selection, add dummy `body.csv|json` and `schema.json` to the working directory.
```

Comment:
Is this RFC final? After our in person discussions yesterday, I wasn't sure if we ended on having a schema.json file created by default. Also, we *don't* create dataset.json, but a user can make their own for advanced options?